includes/class-wc-stripe-payment-method-configurations.php

```diff
@@ -139,12 +139,12 @@ private static function get_payment_method_configuration_from_stripe() {
 	}
 
 	/**
-	 * Get the parent configuration ID.
+	 * Get the WooCommerce Platform payment method configuration id.
 	 *
-	 * @return string|null
+	 * @return string
 	 */
 	public static function get_parent_configuration_id() {
-		return self::get_primary_configuration()->parent ?? null;
+		return WC_Stripe_Mode::is_test() ? self::TEST_MODE_CONFIGURATION_PARENT_ID : self::LIVE_MODE_CONFIGURATION_PARENT_ID;
```

Comment:
QQ: I see we're using this as part of options when creating the payment element, when OC is active. What will happen if my account is not a WooCommerce connected account, and we pass the WooCommerce PMC ID anyways?